_includes/install/php_2.1.md

```diff
@@ -1,2 +1,2 @@
-| 5.5.x | 5.6.0&ndash;5.6.4 | 5.6.5&ndash;5.6.x | 7.0.0, 7.0.1 | 7.0.2 | 7.0.3 | 7.0.4 | 7.0.5 | 7.0.6&ndash;7.0.x | 7.1.x |
+| 5.5.x | 5.6.0&ndash;5.6.4 | 5.6.5&ndash;5.6.x | 7.0.0, 7.0.1 | 7.0.2 | 7.0.3 | 7.0.4 | 7.0.5 | 7.0.6&ndash;7.0.x | 7.1.16+ |
```

Comment:
I din't think this is right... Was this a typo?

Reply:
All fixed up.